docs/reference/builder.md

```diff
@@ -691,8 +691,7 @@ flag, for example `docker build --no-cache`.
 See the [`Dockerfile` Best Practices
 guide](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/build-cache) for more information.
 
-The cache for `RUN` instructions can be invalidated by `ADD` instructions. See
-[below](#add) for details.
+The cache for `RUN` instructions can be invalidated by [`ADD`](#add) and [`COPY`](#copy) instructions. See below for details.
```

Comment:
I think we can remove the `See below for details.` part, as the `ADD` and `COPY` words are now links